src/Rokt-Kit.js

```diff
@@ -74,7 +94,46 @@ var constructor = function () {
         }
     }
 
+    function selectPlacements(options) {
+        var attributes = (options && options.attributes) || {};
```

Comment:
let's add `identifier` to here as well https://docs.rokt.com/developers/integration-guides/web/library/select-placements-options.  It is optional but customers may already be passing this in.

https://docs.rokt.com/developers/integration-guides/web/library/select-placements-options

Reply:
actually i realize that identifier will just be passed through on line 109, right?

Reply:
i would include a test where you are passing `identifier` through and confirm it is passed to `selectPlacements`

Reply:
Yes, `identifier` is passed through. It should be defined in the interfaces for the Rokt Manager